controller/work_item_link_type.go

```diff
@@ -89,8 +91,7 @@ func enrichLinkTypeList(ctx *workItemLinkContext, list *app.WorkItemLinkTypeList
 	for categoryID := range categoryIDMap {
 		modelCategory, err := ctx.Application.WorkItemLinkCategories().Load(ctx.Context, categoryID)
 		if err != nil {
-			jerrors, httpStatusCode := jsonapi.ErrorToJSONAPIErrors(err)
-			return ctx.ResponseData.Service.Send(ctx.Context, httpStatusCode, jerrors)
+			err
```

Comment:
@hector: after looking a little more at the code: the repositories involved in the utility functions returns "qualified errors" (eg: https://github.com/xcoulon/almighty-core/blob/Issue1117_conditional_requests_work_item_link_types/workitem/link/category_repository.go#L61) so we can return the `err` as-is to the controller method which in turn just needs to pass it directly as the 2nd arg to `jsonapi.JSONErrorResponse()` without needing to wrap it using one of the `goa.ErrXYZ()` constructor. Ok for you ?